README.md

```diff
@@ -2,15 +2,18 @@
 
 A new Flutter project. This is a scable directory structrue for Flutter projects.
 
-## Getting Started
 
-This project is a starting point for a Flutter application.
+## Steps to start project
+1. Start the emulator on your machine. If you doesn't have one already created, [create one](https://developer.android.com/studio/run/managing-avds).
+2. Go to root of the project and run `flutter pub get`
+3. Click on the green play button at the toolbar of Android Studio.
+4. Let the application boot and get installed into the emulator.
 
-A few resources to get you started if this is your first Flutter project:
-
-- [Lab: Write your first Flutter app](https://flutter.dev/docs/get-started/codelab)
-- [Cookbook: Useful Flutter samples](https://flutter.dev/docs/cookbook)
-
-For help getting started with Flutter, view our
-[online documentation](https://flutter.dev/docs), which offers tutorials,
-samples, guidance on mobile development, and a full API reference.
+## Connect real device over Wi-Fi
+1. Connect your device with laptop using USB and connect mobile with same WiFi to which your laptop is connect
+2. Open CMD/Terminal
+3. Check if device correctly connected using `adb devices`
+4. Run `adb tcpip 5555`
+5. Open your mobile `Settings` > `About Phone` > `Status` > `IP Address` (IP Address will change everytime you connect)
+6. Run `adb connect <your_mobile_ip>` eg: - `adb connect 192.168.1.6`
```

Comment:
you have to rub "adb connect 192.168.1.6:5555" to connect to device

Reply:
@daksheshvira I never really got a need to use `adb kill-server`,  `adb usb`. Can please explain what these two will actually do?

Though `adb tcpip 5555` is anyway mentioned in step 4.

Reply:
@yashwp "adb kill-server kills" the adb debugging and "adb usb" starts the adb debugging for usb and "adb tcpip 5555" starts the network debugging.

Reply:
@daksheshvira So, can you create a `patch` to this PR?